### ISSUE TITLE
`Alt+[]` also triggered by `Ctrl+PageUp/PageDown`

### ISSUE THREAD

**Issue by jovandeginste** [zellij-org/zellij#2441]
**Basic information**

`zellij --version`: 0.36.0
`stty size`: 58 119
`uname -av` or `ver`(Windows): 5.15.0-71-generic #78-Ubuntu (and other)

List of programs you interact with as, `PROGRAM --version`: output cropped meaningful, for example:
Tilix version: 1.9.5 (but other terminal emulators show the same behaviour)

**Further information**
1. start a fresh instance
2. create multiple panes
3. use `ctrl+pageup` and `ctrl+pagedown`
4. see the layout change as if using `alt+[]`

**Comment by zsehanovic** [comment]
Thought this might help with debugging:

**[**

| Key | ASCII | Keycode | ScanCode |
|------|----|----|----|
|[     |91  |0133|0x5b|

**]**

| Key | ASCII | Keycode | ScanCode |
|------|----|----|----|
|]     |93  |0135|0x5d|


**Alt + [**
| Key | ASCII | Keycode | ScanCode |
|------|----|----|----|
|^[[ 	|27 |0033 |0x1b|
| 	 |91 |0133 |0x5b|




**Alt + ]**

| Key | ASCII | Keycode | ScanCode |
|------|----|----|----|
|^[] 	|27 |0033 |0x1b|
| 	|93 |0135 |0x5d|



**PgUp**

| Key | ASCII | Keycode | ScanCode |
|------|----|----|----|
|^[[5~|27|0033|0x1b|
|        |91|0133|0x5b|
|	 |53|0065| 0x35|
|        |126|0176 |0x7e|




**PgDn**

| Key | ASCII | Keycode | ScanCode |
|------|----|----|----|
|^[[6~|27|0033|0x1b|
|        |91|0133|0x5b|
|	 |54|0066| 0x36|
|        |126|0176 |0x7e|


**C^PgUp**

| Key | ASCII | Keycode | ScanCode |
|------|----|----|----|
|^[[5;5~|27|0033|0x1b|
|          |91|0133|0x5b|
| 	   |53|0065| 0x35|
| 	   |59|0073| 0x3b|
|	   |53|0065| 0x35|
|          |126|0176 |0x7e|


**C^PgDn**

| Key | ASCII | Keycode | ScanCode |
|------|----|----|----|
|^[[6;5~|27|0033|0x1b|
|        |91|0133|0x5b|
|	 |54|0066| 0x36|
|	 |59|0073| 0x3b|
|	 |53|0065| 0x35|
|        |126|0176 |0x7e|

**Comment by TheSast** [comment]
This also happends on `Ctrl/Alt + Insert` and `Ctrl/Alt + Delete`

**Comment by rnovacek** [comment]
`Shift + PageUp` and `Shift + PageDown` have the same effect and cannot be remapped (I have raised #2673 about it).

**Comment by tbottomparallel** [comment]
Also `Alt PageUp` and `Alt PageDown` trigger the layout changes, and cannot be unbound
```
$ zellij
  × Failed to parse Zellij configuration
   ╭─[/home/tbottom/.config/zellij/config.kdl:2:1]
 2 │ keybinds {
 3 │     unbind "Alt PageUp"
   ·     ─────────┬─────────
   ·              ╰── Invalid key: 'Alt PageUp'
 4 │     normal {
   ╰────
  help: For more information, please see our configuration guide: https://zellij.dev/documentation/configuration.html
```

and in a similar vein `Ctrl Alt Arrow` and `Ctrl Shift Arrow` and `Ctrl Alt Shift Arrow` all behave like `Alt Arrow`

**Comment by andreamtp** [comment]
Same problem here: on linux you learn pretty soon to Shift PgUP/PgDown to scroll through the terminal.

If it cannot be mapped to scroll up/ down, at least avoid making it mapped to something else: muscolar memory is hard to remap, but having it triggering other changes is really disappointing.

This bug raise frustration in daily usage of a your great project, please don't forget about user experience of linux nerds, that is likely one of your major base.

**Comment by brianmay** [comment]
Yes, I wouldn't mind if the key combination did nothing. But when it suddenly changes the layout of my panes, which were perfect they way there were. Arrgh. And I can't restore then back again.

Problem is though, I have no idea where this mapping is coming from. Or what function it maps to even. I have tried searching github, but can't find anything relevant.

I think the first step to solving this issue is knowing exactly what happens when you push Shift/Ctrl/Alt PageUp/PageDown.

If somebody could point me to a code reference where these mappings are defined, that would be a huge help.

**Comment by tomasol** [comment]
v0.40.1 is affected as well, however when I built the latest main - 7559b26c - the issue is gone :tada: 
I don't see any `Ctrl PgUp/PgDn`, `Alt PgUp/PgDn` or `Shift PgUp/PgDn` weirdness, and can finally use `Ctrl Insert` in helix.

**Comment by jovandeginste** [comment]
I can confirm that the issues seems to be gone with v0.41.1! :partying_face:

**Comment by brianmay** [comment]
Yes, the built in bindings are gone. Much better.

Wondering if you can create custom bindings for any of these keystrokes now?
